packages/stack/src/views/Stack/CardContainer.tsx

```diff
@@ -55,6 +55,7 @@ type Props = TransitionPreset & {
   headerMode: StackHeaderMode;
   headerShown?: boolean;
   headerTransparent?: boolean;
+  absoluteFloatHeader: boolean;
```

Comment:
Maybe something like `isFloatHeaderAbsolute`. `absoluteFloatHeader` feels like it contains the absolute float header.